### ISSUE TITLE
stripe/stripe-php composer error

### ISSUE THREAD

**Issue by bfjare** [enupal/stripe#82]
Trying to install the plugin locally.  Hangs when trying to add from the control panel.

Run "composer require enupal/stripe" on the command line and get the following errors.

```
Problem 1
    - Installation request for enupal/stripe ^2.5 -> satisfiable by enupal/stripe[2.5.0].
    - Conclusion: remove stripe/stripe-php v6.43.1
    - Conclusion: don't install stripe/stripe-php v6.43.1
    - enupal/stripe 2.5.0 requires stripe/stripe-php ^7.0.0 -> satisfiable by stripe/stripe-php[v7.0.0, v7.0.1, v7.0.2, v7.1.0, v7.1.1, v7.10.0, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.14.2, v7.15.0, v7.16.0, v7.17.0, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.21.1, v7.22.0, v7.23.0, v7.24.0, v7.25.0, v7.26.0, v7.27.0, v7.27.1, v7.27.2, v7.27.3, v7.3.0, v7.3.1, v7.4.0, v7.5.0, v7.6.0, v7.7.0, v7.7.1, v7.8.0, v7.9.0].
    - Can only install one of: stripe/stripe-php[v7.0.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.0.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.0.2, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.1.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.1.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.10.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.11.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.12.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.13.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.14.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.14.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.14.2, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.15.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.16.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.17.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.18.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.19.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.19.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.2.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.2.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.2.2, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.20.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.21.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.21.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.22.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.23.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.24.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.25.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.26.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.27.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.27.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.27.2, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.27.3, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.3.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.3.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.4.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.5.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.6.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.7.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.7.1, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.8.0, v6.43.1].
    - Can only install one of: stripe/stripe-php[v7.9.0, v6.43.1].
    - Installation request for stripe/stripe-php (locked at v6.43.1) -> satisfiable by stripe/stripe-php[v6.43.1].

```

Here is my composer.json file.

```
{
  "require": {
    "aelvan/inlin": "2.1.1",
    "barrelstrength/sprout-email": "4.1.6",
    "craftcms/aws-s3": "1.2.7",
    "craftcms/cms": "3.4.10.1",
    "craftcms/feed-me": "4.2.1.1",
    "craftcms/redactor": "2.6.1",
    "dolphiq/redirect": "1.0.23",
    "dolphiq/sitemap": "1.2.0",
    "enupal/stripe": "2.5.0",
    "jmoont/pdfthumb": "1.0.6",
    "kinoli/groupie": "1.0.3",
    "mikestecker/craft-videoembedder": "1.1.4",
    "nystudio107/craft-minify": "1.2.9",
    "solspace/craft-freeform": "3.6.5",
    "solspace/craft3-calendar": "3.0.2",
    "topshelfcraft/wordsmith": "3.1.2",
    "verbb/comments": "1.3.7",
    "verbb/field-manager": "2.1.1",
    "verbb/super-table": "2.4.6",
    "vlucas/phpdotenv": "^2.4.0"
  },
  "autoload": {
    "psr-4": {
      "modules\\": "modules/"
    }
  },
  "config": {
    "sort-packages": true,
    "optimize-autoloader": true,
    "platform": {
      "php": "7.0"
    }
  },
  "scripts": {
    "post-root-package-install": [
      "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
    ]
  }
}
```

**Comment by andrelopez** [comment]
Thanks for reporting @bfjare We just released Stripe Payments 2.5.1 with a possible fix to this issue, could you please try that and let us know how it goes?

**Comment by bfjare** [comment]
Confirming that this fixed the issue.  Thanks!
